tests/compose/test_compose_sh.py

```diff
@@ -16,16 +16,12 @@
 ## ---------------------------------------------------------------------------
 """Test the compose-sh functionality
 """
-import os
-import pathlib
-
-import pytest
-
-from alyeska.compose.compose_sh import convert_yaml_to_sh
+from alyeska.compose.compose_sh import init_flags, convert_yaml_to_sh
 
 from test_compose_globals import COMPOSE_SMALL, COMPOSE_BIG, COMPOSE_CYCLE
 
 
 def test__convert_yaml_to_sh():
+    init_flags(["foo"])  # Necessary to initialize the flags object.
```

Comment:
Do we want to raise an error if `init_flags` receives an empty list?

Reply:
It already does since config_file is a required argument.